code/ignacio/lab_5a_palindrome.py

```diff
@@ -0,0 +1,33 @@
+# import string
+
+
```

Comment:
try again with these two inputs:

"anagram" and "nag a ram"
"anagram" and "Anagram"

Both should be anagrams. Check the assignment for further missing steps.